frontend/packages/knative-plugin/integration-tests/features/knative/actions-on-knative-revision.feature

```diff
@@ -117,8 +99,6 @@ Feature: Perform actions on knative revision
 
         @smoke @manual
```

Comment:
```suggestion
        @smoke, @manual
```

Reply:
this is expected, or else yarn run gherkin lint will throw the error